### PR TITLE
Added 4 cyber security questions related to security operations 

### PR DESCRIPTION
**Pull request by harryfensome**
(Hacktoberfest)

### CODE REVIEW

**Comment by harryfensome** [comment]
Corrected spelling, UK versions of orgaisation and authorisation and (Hacktoberfest)

File: cybersecurity/cybersecurity-quiz.md [hunk 1]
```diff
@@ -858,3 +858,37 @@ What is the next step you should take to best fulfill your responsibilities and
...
+- [x] Hotfix
+- [ ] Update
+- [ ] Security Fix
+- [ ] Service Pack
+
+#### Q114. How often should security teams conduct a review of the privileged access that a user has to sensitive systems?
+
+- [x] On a periodic basis
+- [x] When a User leaves the organisation
+- [x] When a User changes roles
```
Comment (Ebazhanov) [inline cybersecurity/cybersecurity-quiz.md:882]: we should keep only one 😉 
can you be improved in the next PR

**Review by Ebazhanov — APPROVED** [review]
Looks great, thank you @harryfensome for your valuable contribution! 👍

**Comment by Ebazhanov** [comment]
@all-contributors please add @harryfensome for design, code

**Pull request merged**